### ISSUE TITLE
DeprecationWarning: object() takes no parameters

### ISSUE THREAD

**Issue by walrusVision** [sqlboy/fileseq#71]
On a recent upgrade of pytest, it started returning this warning being emitted from fileseq: `DeprecationWarning: object() takes no parameters`. This was from fileseq-1.7.1 running under python-2.7.15.

To reproduce
```
/virtualenv/bin/python -Wd
Python 2.7.15 (default, Feb 27 2019, 15:50:41)
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import fileseq
>>> fileseq.FrameSet("1-10")
~/virtualenv/lib/python2.7/site-packages/fileseq/frameset.py:83: DeprecationWarning: object() takes no parameters
  self = super(cls, FrameSet).__new__(cls, *args, **kwargs)
FrameSet("1-10")
```

**Comment by justinfx** [comment]
Ah yea. That is coming from the fact that is has changed in py3. This was actually already fixed in a pending branch to introduce py3 support:
https://github.com/sqlboy/fileseq/pull/70

Do you want to give that a test to confirm? Also, I would love to get some feedback from at least one person before deciding how to merge that new version.

**Comment by walrusVision** [comment]
From the branch `63_python3_v2`

```
Python 2.7.15 (default, Feb 22 2019, 04:00:34) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import fileseq
>>> fileseq.FrameSet("1-10")
fileseq/frameset.py:89: DeprecationWarning: object() takes no parameters
  self = super(cls, FrameSet).__new__(cls, *args, **kwargs)
FrameSet("1-10")
```

Also I had a quick try of this branch on 3.6.7 but got this unrelated error
```
Python 3.6.7 (default, Feb 22 2019, 04:02:18)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import fileseq
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/virtualenv/lib/python3.6/site-packages/fileseq/src/fileseq/__init__.py", line 29, in <module>
    from fileseq.frameset import FrameSet
  File "~/virtualenv/lib/python3.6/site-packages/fileseq/src/fileseq/frameset.py", line 13, in <module>
    from fileseq.utils import xfrange, unique, pad
  File "~/virtualenv/lib/python3.6/site-packages/fileseq/src/fileseq/utils.py", line 60, in <module>
    xrange = xrange
NameError: name 'xrange' is not defined
```

**Comment by justinfx** [comment]
Something is fishy about your checkout of that branch. Those line numbers are  not the implementation of that branch:

https://github.com/sqlboy/fileseq/blob/63_python3_v2/src/fileseq/frameset.py#L89
https://github.com/sqlboy/fileseq/blob/63_python3_v2/src/fileseq/utils.py#L60

You must still best testing another branch?
There are CI tests that run this branch against py3 already:
https://travis-ci.org/sqlboy/fileseq/builds/473770716
https://ci.appveyor.com/project/justinfx/fileseq/builds/21302463

**Comment by walrusVision** [comment]
Quite right, I'm not sure what happened there. From a fresh clone and checkout that's working much better

```
~/virtualenv/bin/python -Wd
Python 2.7.15 (default, Feb 22 2019, 04:00:34) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
[273]> import fileseq
[273]> fileseq.FrameSet("1-10")
FrameSet("1-10")
```

**Comment by justinfx** [comment]
Closing as it is fixed by way of #63
